advanced/editor-command-identifiers.md

```diff
@@ -7,93 +7,397 @@ description: The complete list of exposed editor commands.
+| mceInsertLink      | Inserts a link at the current selection. The value is the URL to add to the link(s).                                                                          |
+| Unlink             | Removes any links from the current selection.                                                                                                                 |
+| JustifyLeft        | Left aligns the current text block/image.                                                                                                                     |
+| JustifyCenter      | Center aligns the current text block/image.                                                                                                                   |
+| JustifyRight       | Right aligns the current text block/image.                                                                                                                    |
+| JustifyFull        | Full aligns the current text block/image.                                                                                                                     |
+| JustifyNone        | Removes any alignment to the selected text.                                                                                                                   |
+| ForeColor          | Changes the text color of the text. The value passed in should be the color.                                                                                  |
+| HiliteColor        | Changes the background color of the text. The value passed in should be the color.                                                                            |
+| FontName           | Font name to apply to the text. The value passed in should be the font family name.                                                                           |
+| FontSize           | Font size of the text. The value passed in should be a valid CSS font size.                                                                                   |
+| mceApplyTextcolor  | Applies text color or background color to the current selection. Requires an argument of either `'hilitecolor'` or `'forecolor'`, and the value of the color. |
+| mceRemoveTextcolor | Removes the text color or background color from the current selection. Requires an argument of either `'hilitecolor'` or `'forecolor'`.                       |
+| RemoveFormat       | Removes any formats from the current selection.                                                                                                               |
+| mceBlockQuote      | Wraps the selected text blocks into a block quote.                                                                                                            |
+| FormatBlock        | Changes the format of the current block. The value passed in should be the block name. For example: `H1`, `bold`, `code`, or `P`.                             |
```

Comment:
"selected blocks" instead of "current block"?
But yeah it's probably good that we don't mention that this could take inline formats as well. That's just confusing.

Reply:
Fair, but someone will find it, use it, and ask why it's not documented.

Reply:
Fixed. and the accepted formats have been documented.